worlds/kdl3/docs/setup_en.md

```diff
@@ -16,9 +16,9 @@
 
 ## Installation Procedures
 
-1. Download and install SNIClient from the link above, making sure to install the most recent version.
+1. Download and install Archipelago from the link above, making sure to install the most recent version.
    **The installer file is located in the assets section at the bottom of the version information**.
-    - During setup, you will be asked to locate your base ROM file. This is your Kirby's Dream Land 3 ROM file.
+    - During generation/patching, you will be asked to locate your base ROM file. This is your Kirby's Dream Land 3 ROM file.
```

Comment:
A lot of ROM setup guide give the expected exact name of the rom, I think it would be nice here as well

Reply:
Your call, I just mention what comes to my mind :)

Reply:
When settings receives the file, it'll copy it to the Archipelago directory with the correct name. I think here we're good to just say "It wants your rom" effectively.